packages/core/ui-settings/core-ui-settings-common/src/theme.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const DEFAULT_THEME_NAME = 'amsterdam';
+export const SUPPORTED_THEME_NAMES = ['amsterdam', 'borealis'];
+
+export type ThemeName = (typeof SUPPORTED_THEME_NAMES)[number];
```

Comment:
This isn't actually typed, unless `SUPPORTED_THEME_NAMES` is declared `as const`.

Check out [this playground](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAIgUQGIEECqAZAKgfSwCQQFkEcA5FEmAXhgHIBDAW2gFMAnAE2boG4AoUJFgBlNAAVxAeQBKWBHDyES5SghE0YAbUYsoHbkzoAaegCMQ7VgwA2ASwh0AugP5QAngAdWMLAAtWJlYyZh9aAAoPbxAAMxgxSVl5RQJiUgoSEQBKLTAAVyYzDhd+QXBoGCgAoIAuX2rg0M06YHswKD4YGAB6bvi-EDybTkq-dhAAdxgOcfZSoQqsAE1xBRwE6Tk11JUM9U0dZjYuHlM6CytbB2cYBggYBagBKJ8sL1ZOf0DGoM0X2N8KzWGyS22U6TUIlyBSK7BKZWElXenBwVW+dTe3k+DRCv1oLTaHQEQA) for details.